sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -43,7 +43,7 @@ export const Durations: {
 
 // @public
 export interface ErrorDetail {
-    additionalProperties?: any;
+    additionalProperties?: Record<string, unknown>;
```

Comment:
`Record<string, unknown>` is better than `any` but if we are going to export this to consumers maybe we should explore a narrower type?

Reply:
I think it means it can be any object - if I notice specific patterns on this then maybe i can update this later, but it's from the generated code